### PR TITLE
Update librewolf to v110.0-2

### PR DESCRIPTION
**Pull request by kovmir**
Update.

### CODE REVIEW

**Comment by aclemons** [comment]
Shell variables are not allowed in `.info` files. Could you update these please?

File: network/librewolf/librewolf.info [hunk 1]
```diff
@@ -1,10 +1,10 @@
 PRGNAM="librewolf"
-VERSION="109.0.1"
+VERSION="110.0"
 HOMEPAGE="https://librewolf.net/"
 DOWNLOAD="UNSUPPORTED"
 MD5SUM=""
-DOWNLOAD_x86_64="https://gitlab.com/api/v4/projects/24386000/packages/generic/librewolf/$VERSION-1/LibreWolf.x86_64.AppImage"
-MD5SUM_x86_64="497c3f3ba79510be20c58c4d187c442d"
+DOWNLOAD_x86_64="https://gitlab.com/api/v4/projects/24386000/packages/generic/librewolf/$VERSION-2/LibreWolf.x86_64.AppImage"
```
Comment (aclemons) [inline network/librewolf/librewolf.info:6]: here ^ `$VERSION` needs to be the actual version

Comment (kovmir) [reply network/librewolf/librewolf.info:6]: I have submitted several Slackbuilds with a $VERSION in the download link.
Also, `sbolint` does not complain about it.

Should I still update it?

Comment (kovmir) [reply network/librewolf/librewolf.info:6]: Also, my previous update (about a week ago) of that same Slackbuild is using this variable.

Comment (aclemons) [reply network/librewolf/librewolf.info]: > Should I still update it?

Yes, it needs to be changed. While info files are valid shell code, not all tools process them that way. Indeed, the components for the sbo db do not. The link for your current submission is broken:

https://slackbuilds.org/repository/15.0/network/librewolf/

> Also, my previous update (about a week ago) of that same Slackbuild is using this variable.

That one should not have made it through

**Comment by aclemons** [comment]
```
# buildpkg x86_64 network/librewolf
sbolint: librewolf checks out OK
git repo: /var/lib/repo/SBo/slackbuilds
branch:   user/aclemons/updates
date:     Tue Feb 28 21:36:17 GMT 2023
revision: 0c2c0ad8e67184c701485fc773f7412007aac232
title:    Update librewolf to v110.0-2

================================================================================
network/librewolf                                                       21:39:26
================================================================================

Calculating dependencies ... none.

Building network/librewolf (add version 110.0) ------------------------ 21:39:26
Testing SlackBuild files ... done.
Downloading source files ...
Verifying source files ... done.
Running librewolf.SlackBuild ...                                    ETA 21:41:??
env MAKEFLAGS='-j24 -l13' nice -n 5 bash ./librewolf.SlackBuild
Build time 84 sec, peak load 1.02, peak memory 1336M, peak tmp 229M
Built ok:  librewolf-110.0-x86_64-1_SBo.txz                             21:40:51
Testing package librewolf-110.0-x86_64-1_SBo.txz ... done.
Test installing librewolf-110.0-x86_64-1_SBo.txz ...
librewolf-110.0-x86_64-1_SBo: Private Web Browser ...................... [ 230M]
Unmounting chroot ... done.
:-) network/librewolf Added version 110.0 (Private Web Browser) (-:

================================================================================
SUMMARY                                                                 21:41:06
================================================================================

Logfile:     /var/log/repo/SBo/repo_2023-02-28_21:39:26.log
Runtime:     0h 1m 40s
Added:       1
  network/librewolf/librewolf-110.0-x86_64-1_SBo.txz

Using tests from /usr/share/sbo-maintainer-tools/sbopkglint.d
Exploding /var/lib/repo/SBo/packages/15.0/x86_64/network/librewolf/librewolf-110.0-x86_64-1_SBo.txz to /tmp/sbopkglint.WIADal ...OK
Running pre-doinst test...OK
Installing /var/lib/repo/SBo/packages/15.0/x86_64/network/librewolf/librewolf-110.0-x86_64-1_SBo.txz to /tmp/sbopkglint.WIADal ...OK
Running test: 05-basic-sanity...OK
Running test: 10-docs...OK
Running test: 15-noarch...OK
Running test: 20-arch...OK
Running test: 25-lafiles...OK
Running test: 30-manpages...OK
Running test: 35-desktop...OK
Running test: 40-newconfig...OK
Running test: 45-doinst...OK
Running test: 50-icons...OK
=== librewolf-110.0-x86_64-1_SBo.txz: All tests passed
```

**Comment by aclemons** [comment]
LGTM - staged to my branch

**Pull request closed without merging**